File: fe_flutter_ui/lib/models/notification.dart
class Notifications {
  int? id;
  String? userId;
  String? title;
  String? body;
  String? createdAt;
  bool? status;

  Notifications({this.id, this.userId, this.title, this.body, this.createdAt, this.status});

  Notifications.fromJson(Map<String, dynamic> json) {
    id = json['id'];
    userId = json['userId'];
    title = json['title'];
    body = json['body'];
    createdAt = json['createdAt'];
    status = json['status'];
  }

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = <String, dynamic>{};
    data['id'] = id;
    data['userId'] = userId;
    data['title'] = title;
    data['body'] = body;
    data['createdAt'] = createdAt;
    data['status'] = status;
    return data;
  }
}

File: be_aspnet_api/MrSoaiAPI/Model/Notification.cs
using System.ComponentModel;

namespace MrSoaiAPI.Model
{
    public class Notification
    {
        public int Id { get; set; }
        public string UserId { get; set; }
        public string Title { get; set; }
        public string Body { get; set; }
        public DateTime CreatedAt { get; set; }
        [DefaultValue(true)]
        public bool Status { get; set; } = true;
    }
}
